util/uid.go

```diff
@@ -0,0 +1,3 @@
+package util
+
+type Uid uint
```

Comment:
:+1: big fan of this sort of thing

Reply:
Yeah, it makes messing things up a lot harder lol